### DISCUSSION TITLE
Main page with 2 vehicles

### DISCUSSION THREAD

**Discussion by natrlhy** [teslamate-org/teslamate#2351] [Q&A]
We will be getting a 2nd Tesla soon and I was curious what the main page looks like with 2 cars. Is anyone able to share a screenshot of what it looks like? Are both shown at the same time if you scroll up or down?

Also when the 2nd car is added to the same Tesla account do I still have to stop/start TeslaMate to pick up the 2nd car so it starts to log data for it?

Thanks!

**Comment by LukeSkywalker993** [comment]
Hello, the second car behaves exactly like the first car. So you have the exact same window with the map and status as a box underneath.
It is important that when you get the second car, you first open the Tesla app and then restart the Contrainer.

**Reply by natrlhy** [reply]
Thanks! I'll make sure I add this to my checklist before driving it away for the first time. 🚙
